config.sample.yaml

```diff
@@ -321,6 +321,9 @@ ircService:
     # on instead of 113 here.
     # Default: 113.
     port: 1113
+    # The address to listen on for incoming ident requests.
+    # Default: 0.0.0.0
+    address: "::"
```

Comment:
If you run this on a machine which does not support IPv6, does this work? I suspect it will fall over, so I would prefer if this value were set to `"0.0.0.0"` instead. The [docs](https://nodejs.org/api/net.html#net_server_listen_port_hostname_backlog_callback) aren't super clear either way.